.slim-lint.yml

```diff
@@ -0,0 +1,41 @@
+exclude:
+  - '.gems/**/*.slim'
+  - 'vendor/**/*.slim'
+  - 'app/views/layouts/application.html.slim'
```

Comment:
I think is important to check `application.html.slim`. I suggest you move the exact parts with offences to partials and exclude only those partials.